packages/api/src/graphql/memberPlan.ts

```diff
@@ -0,0 +1,110 @@
+import {AvailablePaymentMethod, MemberPlan, MemberPlanSort} from '../db/memberPlan'
+
+import {GraphQLRichText} from './richText'
+import {GraphQLImage} from './image'
+import {Context} from '../context'
+import {createProxyingResolver} from '../utility'
+import {
+  GraphQLID,
+  GraphQLNonNull,
+  GraphQLObjectType,
+  GraphQLString,
+  GraphQLBoolean,
+  GraphQLList,
+  GraphQLInt,
+  GraphQLInputObjectType,
+  GraphQLEnumType
+} from 'graphql'
+import {GraphQLDateTime} from 'graphql-iso-date'
+import {GraphQLPageInfo} from './common'
+import {GraphQLPaymentMethod} from './paymentMethod'
+
+export const GraphQLAvailablePaymentMethod = new GraphQLObjectType<AvailablePaymentMethod, Context>(
+  {
+    name: 'AvailablePaymentMethod',
+    fields: {
+      paymentMethods: {
+        type: GraphQLNonNull(GraphQLList(GraphQLNonNull(GraphQLPaymentMethod))),
+        async resolve({paymentMethodIDs}, args, {dbAdapter}) {
+          const paymentMethods = await dbAdapter.paymentMethod.getPaymentMethods()
+          return paymentMethods.filter(paymentMethod => paymentMethodIDs.includes(paymentMethod.id))
+        }
+      },
+      paymentPeriodicities: {type: GraphQLNonNull(GraphQLList(GraphQLNonNull(GraphQLString)))},
+      forceAutoRenewal: {type: GraphQLNonNull(GraphQLBoolean)}
+    }
+  }
+)
+
+export const GraphQLMemberPlan = new GraphQLObjectType<MemberPlan, Context>({
+  name: 'MemberPlan',
+  fields: {
+    id: {type: GraphQLNonNull(GraphQLID)},
+
+    createdAt: {type: GraphQLNonNull(GraphQLDateTime)},
+    modifiedAt: {type: GraphQLNonNull(GraphQLDateTime)},
+
+    name: {type: GraphQLNonNull(GraphQLString)},
+    image: {
+      type: GraphQLImage,
+      resolve: createProxyingResolver(({imageID}, args, {loaders}) => {
+        return imageID ? loaders.images.load(imageID) : null
+      })
+    },
+    description: {type: GraphQLRichText},
+    isActive: {type: GraphQLNonNull(GraphQLBoolean)},
+    pricePerMonthMinimum: {type: GraphQLNonNull(GraphQLInt)},
```

Comment:
Same q about a price being whole number

Reply:
GraphQLInt ist an integer. I don't understand you question. :)

Reply:
@roosnic1 I would assume the default for all pricing to be a fraction number, not whole number unless it was a deliberate choice to only use round numbers payments.